api/proto/teleport/legacy/types/types.proto

```diff
@@ -5102,8 +5102,8 @@ message PluginSlackAccessSettings {
 message PluginOpsgenieAccessSettings {
   option (gogoproto.equal) = true;
 
-  // Addr is the address of Opsgenie API.
-  string addr = 1;
+  // APIEndpoint is the address of Opsgenie API.
+  string api_endpoint = 1;
```

Comment:
This breaks JSON marshaling and unmarshaling of `PluginOpsgenieAccessSettings`, both in the `encoding/json` sense and in the `protojson` sense. If the break was deliberate then remove the addr field with number 1, add `reserved 1; reserved "addr";` and add `api_endpoint` with number 5. to the message.

Reply:
@espadolini The `PluginOpsgenieAccessSettings` message is present in v13 but it is not used by anything yet. Is renaming this field still an issue in this case?

Reply:
It's still very bad practice in general, and why add to the mental load of anyone who is going to look at this change? Field numbers and field names are free.